db/migrate/20210910120352_create_comments.rb

```diff
@@ -0,0 +1,13 @@
+class CreateComments < ActiveRecord::Migration[6.1]
+  def change
+    create_table :comments do |t|
```

Comment:
no soft delete option in the tables. records will be deleted permanently.
https://github.com/ActsAsParanoid/acts_as_paranoid  link for reference .add to each of the model